Gems/ImGui/Code/Include/LYImGuiUtils/HistogramGroup.h

```diff
@@ -36,6 +36,8 @@ namespace ImGui::LYImGuiUtils
 
         void SetHistogramBinCount(int count) { m_histogramBinCount = count; }
 
+        ImGui::LYImGuiUtils::HistogramContainer* FindContainerByName(const char* name);
```

Comment:
nit: Looks like this could be `const`

Reply:
Resolving: We talked offline about this, resolving as that would introduce a const_cast<> inside the method.

Reply:
Yes sorry by bad, returning a reference to non-const data totally makes this the right call (could have a `const` overload in future if need be 👍)